### DISCUSSION TITLE
Github Pages Cloning

### DISCUSSION THREAD

**Discussion by irrevocablesake** [community/community#22122] [Other Feature Feedback, Questions, & Ideas]
<p>I recently started a static blog and hosted it on github pages, added google analytics to it too. I have implemented some programming projects and written blog post on it and hosted the code on github.</p>
<p>Now, the thing is at least there are over like 9-15 clones per day for the hosted code and the code for my website my website is only like 30-34 days old(I posted about the blog on reddit).</p>
<p>I do not understand why would people clone my website code this early or are these bots?.</p>
<p>Thanks.</p>

**Comment by BaileyJM02** [comment]
<p>If you are referring to the git clones under the <a href="https://github.com/irrevocablesake/irrevocablesake.github.io/graphs/traffic" rel="nofollow noopener">Insights/Traffic</a> tab, this may be your editor, such as VSCode, checking that your local repository matches the one on GitHub. I’m sure it’s nothing to worry about. If you are worried about people ‘stealing’ your content, you could always create a <a href="https://docs.github.com/en/enterprise/2.16/user/github/getting-started-with-github/create-a-repo" rel="nofollow noopener">private repository</a>.</p>

**Comment by irrevocablesake** [comment]
<p><a class="mention" href="/baileyjm02">@baileyjm02</a></p>
<p>Thanks for the reply.</p>
<p>I do use vscode as my code editor with the live git tracking feature. I am not worried about people stealing about my content ( In fact i am okay with people downloading the exe and code from my github, my blog is kind of a walkthrough, so that they can make something good with it) but i just wanted to check how many people visit my github.</p>
<p>I was treating the insights/traffic tab as a perfect metric system for the number of visitors to my github and number of people who cloned my programs. I do have google analytics but again that does not help with github visiting and cloning metrics.</p>
<p>Is there any extension that would help me track the number of people who actually visited and cloned the repos.</p>

**Comment by BaileyJM02** [comment, marked as answer]
<p>Yeah, the GitHub insights should be taken more as a guide, <em>however</em>, the actual repository visits do seem to be more accurate. Try looking at the <strong>unique</strong> values, they will probably paint a more realistic picture.</p>
<p>Regarding an extension for tracking the number of people visiting your GitHub, there aren’t any, I’m afraid the repository insights are all you have.</p>

**Comment by irrevocablesake** [comment]
<p>Alright Thanks <a class="mention" href="/baileyjm02">@baileyjm02</a> i appreciate your help.</p>
